### DISCUSSION TITLE
Check the recordings of our weekly community call!

### DISCUSSION THREAD

**Discussion by xiaoya-yaya** [hypertrons/hypertrons-crx#44] [General]
## Hey there!

We have weekly sync-ups each **Wednesday**. If you miss it, check our discussions in the meeting minutes:
[https://shimo.im/docs/tH9JQXWTXqgPypPw](https://shimo.im/docs/tH9JQXWTXqgPypPw)

Or watch the video, we will keep the list updated here:

### March 2021

**March 3** -- [https://www.bilibili.com/video/BV1D54y1h7rM/](https://www.bilibili.com/video/BV1D54y1h7rM/)
**March 10** -- https://www.bilibili.com/video/BV1Xp4y1H7hm/
**March 17** -- https://www.bilibili.com/video/BV18K4y1U73R/
**March 24** -- https://www.bilibili.com/video/BV1d5411w7Fe/

### April 2021

**April 7** -- https://www.bilibili.com/video/BV1vB4y1P7zJ/
**April 13** -- https://www.bilibili.com/video/BV1qK4y1K7yU/
**April 21** -- https://www.bilibili.com/video/BV1yv411773L/
**April 28** -- https://www.bilibili.com/video/BV1e64y1d7hA/

### May 2021

**May 12** -- https://www.bilibili.com/video/BV19v4115736/
**May 19** -- https://www.bilibili.com/video/BV1zB4y1F7do/

**Comment by heming6666** [comment]
Great! Thank you @xiaoya-Esther

**Comment by heming6666** [comment]
March 10 -- https://www.bilibili.com/video/BV1Xp4y1H7hm/

**Comment by heming6666** [comment]
March 17 -- https://www.bilibili.com/video/BV18K4y1U73R/

**Comment by heming6666** [comment]
March 24 -- https://www.bilibili.com/video/BV1d5411w7Fe/
